doc/api/test.md

```diff
@@ -546,6 +546,16 @@ Node.js, and should not be relied on programmatically. If programmatic access
 to the test runner's output is required, use the events emitted by the
 {TestsStream}.
 
+The reporters are available via the `node:test/reporters` module:
+
+```mjs
+import { tap, spec, dot } from 'node:test/reporters';
```

Comment:
Can you also update line 522, and add `changes:` with a PR-URL, description etc.

Reply:
@anonrig I'd add it happily, just not sure what version I should write this down as 🙂

Reply:
The version should be REPLACEME.